client/rollup.config.js

```diff
@@ -71,6 +71,7 @@ export default {
         })
     ],
 
+    context: 'this',
```

Comment:
d'après https://stackoverflow.com/questions/43556940/rollup-js-and-this-keyword-is-equivalent-to-undefined permet d'eviter d'avoir les warnings sur le "this undefined" dans la compilation client

Reply:
Super ça !